simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xPerfMedium.scala

```diff
@@ -0,0 +1,500 @@
ite_ruby/2.4.0","rubyversion":"2.4.1","selinux":true,"selinux_config_mode":"enforcing","selinux_current_mode":"enforcing","selinux_enforced":true,"selinux_policyversion":"28","serialnumber":"USE346L3KJ","service_provider":"systemd","ssh":{"ecdsa":{"fingerprints":{"sha1":"SSHFP 3 1 7675cdd44d77b0ad1c5999f0a9db18e3aad3cb0d","sha256":"SSHFP 3 2 b4224f6ab000d73c283a4e4330c642bd848910e7fbca0b39de712400d132d193"},"key":"AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCGU9YUZnsIvwTZd59H1txba3096BhpHRf0l7QDupFgXqwWGcRr1Cc4jpYEdAIRfqjmFD3JVQlSa4C0zugGd6Jk="},"ed25519":{"fingerprints":{"sha1":"SSHFP 4 1 25583719f4902d02bebd102f2ab6843e8f993b37","sha256":"SSHFP 4 2 27fec0970e79f7875be3c87a2be118c9b8cdb64f05f6ae5a8aeec7cb4b2bcebc"},"key":"AAAAC3NzaC1lZDI1NTE5AAAAIIJci3r0YxHyEZyNnqKggerbWshfeXERucZHwH0zDeaf"},"rsa":{"fingerprints":{"sha1":"SSHFP 1 1 b7caf8ce4a295676009b49c012bae99f37bdfd1a","sha256":"SSHFP 1 2 1d645d15c047ac7a3e908f0bd7d6dc23b59f352182c126816236e9c055ac7ebf"},"key":"AAAAB3NzaC1yc2EAAAADAQABAAABAQCn3YlHJ1y9vQxCijwF3MpKg4t3uMwFRFHWSeBxvI7Qh1Uo6rqPogn/5w5SaNnQrMISifadoOHz0Sg+5ZwLUQDpMxSbS6kx0BWr6ZLiDlk8nu58A3njCCSwmTJjMoqoRxQLoCFq6ECYs81Vu8UFUbtgujuSSEuoCGxYrI4j/zMIT3iZnhI69esYk/NuK7SxKfECUwXDRsCArVqaGNpqd56/uOHmjg+8xgSko1AypCGh+lx6Ghg5F7bXyQRsRFlXtvv2h9Fjv9lIwidSNneA14HfAMfcT07VE6sLnwUONHAD4X52iNxarAa/bAQxhqUVvlH/7RN5SaV/aDKZEURXiPEb"}},"sshecdsakey":"AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCGU9YUZnsIvwTZd59H1txba3096BhpHRf0l7QDupFgXqwWGcRr1Cc4jpYEdAIRfqjmFD3JVQlSa4C0zugGd6Jk=","sshed25519key":"AAAAC3NzaC1lZDI1NTE5AAAAIIJci3r0YxHyEZyNnqKggerbWshfeXERucZHwH0zDeaf","sshfp_ecdsa":"SSHFP 3 1 7675cdd44d77b0ad1c5999f0a9db18e3aad3cb0d\nSSHFP 3 2 b4224f6ab000d73c283a4e4330c642bd848910e7fbca0b39de712400d132d193","sshfp_ed25519":"SSHFP 4 1 25583719f4902d02bebd102f2ab6843e8f993b37\nSSHFP 4 2 27fec0970e79f7875be3c87a2be118c9b8cdb64f05f6ae5a8aeec7cb4b2bcebc","sshfp_rsa":"SSHFP 1 1 b7caf8ce4a295676009b49c012bae99f37bdfd1a\nSSHFP 1 2 1d645d15c047ac7a3e908f0bd7d6dc23b59f352182c126816236e9c055ac7ebf","sshrsakey":"AAAAB3NzaC1yc2EAAAADAQABAAABAQCn3YlHJ1y9vQxCijwF3MpKg4t3uMwFRFHWSeBxvI7Qh1Uo6rqPogn/5w5SaNnQrMISifadoOHz0Sg+5ZwLUQDpMxSbS6kx0BWr6ZLiDlk8nu58A3njCCSwmTJjMoqoRxQLoCFq6ECYs81Vu8UFUbtgujuSSEuoCGxYrI4j/zMIT3iZnhI69esYk/NuK7SxKfECUwXDRsCArVqaGNpqd56/uOHmjg+8xgSko1AypCGh+lx6Ghg5F7bXyQRsRFlXtvv2h9Fjv9lIwidSNneA14HfAMfcT07VE6sLnwUONHAD4X52iNxarAa/bAQxhqUVvlH/7RN5SaV/aDKZEURXiPEb","stage":"prod","staging_http_get":"curl","swapfree":"1000.00 MiB","swapfree_mb":999.99609375,"swapsize":"1000.00 MiB","swapsize_mb":999.99609375,"system_uptime":{"days":0,"hours":0,"seconds":1296,"uptime":"0:21 hours"},"timezone":"PDT","uptime":"0:21 hours","uptime_days":0,"uptime_hours":0,"uptime_seconds":1296,"uuid":"37323233-3135-5553-4533-34364C334B4A","virtual":"physical","whereami":"portland","clientcert":"${node}","clientversion":"5.0.0","clientnoop":false},"timestamp":"2017-06-14T17:17:27.391789175-07:00","expiration":"2017-06-14T17:47:27.392052206-07:00"}""")
```

Comment:
In the most recent Puppet 5 agent code, the fact payload now has an extra layer of URL encoding done, as mentioned in https://github.com/puppetlabs/puppet/pull/5999.  It would seem good to just re-record this with the most recent Puppet 5 agent code before applying the other request header edits from above.